### ISSUE TITLE
expected time_distributed_9 to have 3 dimensions, but got array with shape (102, 1)

### ISSUE THREAD

**Issue by rdulmina** [keras-team/keras#11188]
I'm getting the following error when i try to implement keras LSTM model with Bidirectional wrapper.
Here is my code.I have printed the shapes of the arrays bellow.Please help :-(

![image](https://user-images.githubusercontent.com/26381349/45876508-6d4a0600-bdb8-11e8-82ab-d10eb6ff5e62.png)
![image](https://user-images.githubusercontent.com/26381349/45876834-87381880-bdb9-11e8-8954-61dfe2ca25d6.png)

![image](https://user-images.githubusercontent.com/26381349/45876866-a3d45080-bdb9-11e8-8035-be5d3b355f04.png)

**Comment by farizrahman4u** [comment]
Two issues with your code:

1) Your model outputs sequences, but your data Y (`train_agg_list_np`) is not a sequence. Solution: you probably don't need `return_sequences=True` for the final lstm. Also remove the `TimeDistributed`  wrapper, since we are not working with sequences at that point.

2) You are trying to predict a probability over 6 classes, but your data seems labels. Convert them to one-hots using `keras.np_utils.to_categorical`.

P.S post code snippets instead of screenshots next time.

**Comment by rdulmina** [comment]
thank you so much my model is now training :-)
